core/src/test/scala/org/apache/spark/sql/delta/DeltaWriteConfigsSuite.scala

```diff
@@ -82,7 +82,7 @@ class DeltaWriteConfigsSuite extends QueryTest
 
     println("DeltaTableBuilder Test Output")
     dtb_output.toSeq
-      .toDF("Output Location", "Output Mode", s"Contains No-Prefix Option (lowercase)",
```

Comment:
there is a mismatch here. here you have deleted the column `"Contains No-Prefix Option (lowercase)"` yet it is still present in the table below.

I'm fine with deleting this column, but make the output table (commented below) reflect this change.